crates/bevy_diagnostic/src/system_information_diagnostics_plugin.rs

```diff
@@ -46,6 +45,12 @@ pub struct SystemInfo {
     pub memory: String,
 }
 
+/// The expected interval at which system information will be queried and generated.
+///
+/// The system diagnostic plugin doesn't work in all situations. In those situations this value will
+/// bet set to None.
+pub use internal::EXPECTED_SYSTEM_INFORMATION_INTERVAL;
```

Comment:
Why does this need to be exposed?

Reply:
It's mostly a way to expose the fact that system diagnostics will be generated somewhat slowly.

Reply:
Oh, in that case I'd prefer if that was documented directly on the plugin docs so it's a bit more visible. I mean, this can stay here, but it should also be documented on the plugin